lms/services/ltia_http.py

```diff
@@ -15,27 +15,6 @@ def __init__(
         self._jwt_service = jwt_service
         self._http = http
 
-    def sign(self, payload: dict):
-        """
-        Sign a payload with one of our private keys.
-
-        We include the default values needed for LTIA APIs
-        Those default claims are defined here:
-
-            https://www.imsglobal.org/spec/security/v1p0/#id-token
```

Comment:
The comment is correct, it's using those defaults but that's not what we want all the time.